### PR TITLE
feat(grpc): add Ed25519 to AddressType proto

### PR DESCRIPTION
**Pull request by maxipaz**
## Description

Adding Ed25519 to `AddressType` proto

## Related issue(s)

- https://github.com/pactus-project/pactus/issues/1488

### CODE REVIEW

File: www/grpc/proto/wallet.proto [hunk 1]
```diff
@@ -55,6 +55,9 @@ enum AddressType {
   ADDRESS_TYPE_VALIDATOR = 1;
   // Account address type with BLS signature scheme.
   ADDRESS_TYPE_BLS_ACCOUNT = 2;
+  // Account address type with Ed25519 signature scheme.
+  // Note: Generating a new Ed25519 address requires the wallet password.
+  ADDRESS_TYPE_ED25519_ACCOUNT = 3;
```
Comment (Ja7ad) [inline www/grpc/proto/wallet.proto:60]: @b00f For this type needs to get wallet password in the message `GetNewAddressRequest`?

Comment (b00f) [reply www/grpc/proto/wallet.proto:60]: Yes, We need the password to derive a new address. The reason is explained  here: https://github.com/pactus-project/pactus/blob/0fd3ac211075b48abf8315e3a51d757c762066b3/crypto/ed25519/hdkeychain/extendedkey.go#L70

Comment (maxipaz) [reply www/grpc/proto/wallet.proto:60]: Added the password to the `GetNewAddress` input. Please review again

**Review by Ja7ad — APPROVED** [review]

**Comment by Ja7ad** [comment]
@maxipaz Thank you

**Pull request merged**